cst438-project-airline/src/main/java/cst438/domain/SeatRepository.java

```diff
@@ -0,0 +1,7 @@
+package cst438.domain;
+
+import org.springframework.data.jpa.repository.JpaRepository;
+
+public interface SeatRepository extends JpaRepository<Seat, Long> {
+
```

Comment:
We'll need a query of available seats for a given flight and whether it's first class or not.
`select * from seat where flight_id = ? and available = ? and is_first_class = ?;`